user-interface/src/search-results/SearchResults.tsx

```diff
@@ -28,36 +20,73 @@ export function isValidSearchPredicate(searchPredicate: CasesSearchPredicate): b
   }, false);
 }
 
+export function sortByDateFiled(a: CaseBasics, b: CaseBasics): number {
```

Comment:
These feel like functions that will eventually migrate out of this file

Reply:
agreed